pom.xml

```diff
@@ -596,6 +601,11 @@
                 <version>1.7.10</version>
             </dependency>
             <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-log4j12</artifactId>
+                <version>1.7.25</version>
```

Comment:
Is it right for `slf4j-log4j12` to be a different version to `slf4j-api`?  Are these compatible with each other?